db/migrate/20210325162125_create_experiments.rb

```diff
@@ -0,0 +1,14 @@
+class CreateExperiments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :experiments, id: :uuid do |t|
+      t.string :lookup_name, null: false
```

Comment:
This could just be name?

Reply:
Totally. In fact, it was name until yesterday. I changed it to make clear that the name is primarily to enable easier lookups. But I'm happy with name.